src/dotnet-uninstall/Messages.cs

```diff
@@ -34,8 +34,11 @@ internal static class Messages
         public static readonly string UninstallVerbosityOptionArgumentName = "VERBOSITY-LEVEL";
         public static readonly string UninstallVerbosityOptionArgumentDescription = "The specified MSBuild verbosity level";
 
-        public static readonly string SdkOptionDescription = "Uninstall the .NET Core SDKs only";
-        public static readonly string RuntimeOptionDescription = "Uninstall the .NET Core Runtimes only";
+        public static readonly string SdkOptionDescription = "Uninstall .NET Core SDKs only";
```

Comment:
you should start to convert to resx now. The longer you delay that, the harder it gets

Reply:
I agree. But my concern is that there are always pending PRs. I will probably do that during the weekend if we get all merged in today.